packages/react-on-rails-pro/src/cache/index.stub.ts

```
@@ -51,3 +51,38 @@ export function unstable_cache<TArgs extends unknown[]>(
 }
+
+export type { RedisCacheHandlerOptions } from './RedisCacheHandler.ts';
+export type { TieredCacheHandlerOptions } from './TieredCacheHandler.ts';
+
+export class RedisCacheHandler implements CacheHandler {
+  constructor(_options?: unknown) {
+    throw new Error(STUB_ERROR);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/require-await
+  async get(_key: string): Promise<CacheEntry | null> {
```

ESLint

[error] Expected 'this' to be used by class async method 'get'.

(class-methods-use-this)